### ISSUE TITLE
Can support custom-written baud rate?

### ISSUE THREAD

**Issue by MrDuan666** [mich-w/QtSerialMonitor#5]
The baud rate used in my project is not listed in the default list. What do I need to do if I want to set a specific baud rate?

**Comment by mich-w** [comment]
Hmm. Haven't thought about it. The baudrates listed in the comboBox are based on supported values reported by host's hardware - that list is dynamically populated on startup. AFAIK using a non standard baud-rate is highly unusual. 

One could theoretically set that combobox as editable. Then you would simply type in the costom baud-rate if its not listed. But honestly I never tried it. I usually just go with 115200 or 9600. 

Mind if I ask what kind of baudrate are you using ?

**Comment by MrDuan666** [comment]
Yes, you are right. In fact, the app of serial port assistant (Microsoft store) do it as you said, it allows to set a custom baud rate.
 My project always use some large baud rates, such as 1152000 or 2000000.
